admin/Default/enable_game.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+$template->assign('PageTopic', 'Enable New Games');
+
+// If we have just forwarded from the processing file, pass its message.
+$template->assign('ProcessingMsg', $var['processing_msg']);
+
+// Get the list of disabled games
+$db->query('SELECT game_name, game_id FROM game WHERE enabled=' . $db->escapeBoolean(false));
+$disabledGames = array();
+while ($db->nextRecord()) {
+	$disabledGames[] = array('game_name' => $db->getField('game_name'),
+	                         'game_id' => $db->getInt('game_id'));
+}
+$template->assign('DisabledGames', $disabledGames);
+
+// Create the link to the processing file
+$link_container = create_container('enable_game_processing.php', '');
```

Comment:
Can you use `camelCase` variables please - just to try to keep it standardised throughout the code base as much as possible (although with php that's never going to be fully possible as it has multiple styles for built in stuff as it is :(  )